aliyun-ecs/src/test/java/org/jclouds/aliyun/ecs/compute/internal/BaseECSComputeServiceApiLiveTest.java

```diff
@@ -16,17 +16,31 @@
  */
 package org.jclouds.aliyun.ecs.compute.internal;
 
+import com.google.common.base.Predicate;
 import com.google.inject.Injector;
+import com.google.inject.Key;
 import com.google.inject.Module;
+import com.google.inject.TypeLiteral;
+import com.google.inject.name.Names;
 import org.jclouds.aliyun.ecs.ECSComputeServiceApi;
+import org.jclouds.aliyun.ecs.domain.internal.Regions;
 import org.jclouds.apis.BaseApiLiveTest;
 import org.jclouds.compute.config.ComputeServiceProperties;
 
 import java.util.Properties;
 import java.util.concurrent.TimeUnit;
 
+import static org.jclouds.compute.config.ComputeServiceProperties.TIMEOUT_NODE_RUNNING;
+import static org.jclouds.compute.config.ComputeServiceProperties.TIMEOUT_NODE_SUSPENDED;
+
 public class BaseECSComputeServiceApiLiveTest extends BaseApiLiveTest<ECSComputeServiceApi> {
 
+   protected static final String TEST_REGION = Regions.EU_CENTRAL_1.getName();
```

Comment:
Make this readable form a property that can be passed int he command-line.

Reply:
ok done